couchbase/couchbase.go

```diff
@@ -0,0 +1,86 @@
+package couchbase
+
+import (
+	"time"
+
+	"github.com/couchbase/gocb/v2"
+)
+
+type Storage struct {
+	cb     *gocb.Cluster
+	bucket *gocb.Bucket
+}
+
+func New(config ...Config) *Storage {
+	// Set default config
+	cfg := configDefault(config...)
+
+	cb, err := gocb.Connect(cfg.Host, gocb.ClusterOptions{
+		Authenticator: gocb.PasswordAuthenticator{
+			Username: cfg.Username,
+			Password: cfg.Password,
+		},
+		TimeoutsConfig: gocb.TimeoutsConfig{
+			ConnectTimeout: cfg.ConnectionTimeout,
+			KVTimeout:      cfg.KVTimeout,
+		},
+		Transcoder: gocb.NewLegacyTranscoder(),
+	})
+
+	if err != nil {
+		panic(err)
+	}
+
+	_, err = cb.Ping(&gocb.PingOptions{
+		Timeout: 5 * time.Second,
+	})
+
+	if err != nil {
+		panic(err)
+	}
+
+	b := cb.Bucket(cfg.Bucket)
+
+	return &Storage{cb: cb, bucket: b}
+}
+
+func (s *Storage) Get(key string) ([]byte, error) {
+	out, err := s.bucket.DefaultCollection().Get(key, nil)
+	if err != nil {
+		return nil, err
```

Comment:
Is it returning error when the key is not found? Returning nil seems better https://github.com/gofiber/storage/blob/main/storage.go#L10

Reply:
That's appropriate way, it was in my mind I forgot to add 👍